contributions/catalog/controllers/config.py

```diff
@@ -1,43 +1,24 @@
 import os
+from dotenv import load_dotenv
 
 
+# Load .env file
+load_dotenv()
+
 class Config(object):
-    CONTRIBUTION_BUILDING_BLOCK_URL = os.getenv("CONTRIBUTION_BUILDING_BLOCK_URL",
-                                                "http://localhost:5000/contributions")
+    OAUTHLIB_INSECURE_TRANSPORT = True
```

Comment:
OAUTHLIB_INSECURE_TRANSPORT should also be obtained and calculated from an environment variable. The default value should be `False`, as this is only needed for development purposes. Thanks.